src/wp-content/themes/twentytwentyone/functions.php

```diff
@@ -27,6 +27,7 @@
 	 * @return void
 	 */
 	function twenty_twenty_one_setup() {
+		add_theme_support( 'block-template-parts' );
```

Comment:
```suggestion
```

This was only added for testing purposes, correct?

Reply:
Sorry, I forgot to turn it off 🙈